### ISSUE TITLE
in UEFI-Shell，can not run ls cmd under NTFS partition

### ISSUE THREAD

**Issue by Rainzhouzzz** [zhaoyichanghong/Ntfs_For_UEFI#1]
when I load NtfsDxe.efi, the NTFS partition is mounted in UEFI shell, but when i run ls command ,it can not work.Is this normal?
![ls](https://user-images.githubusercontent.com/49701237/77895851-f7c2ce00-72a9-11ea-9b67-ed8050ab999d.png)

**Comment by Rainzhouzzz** [comment]
![ls1](https://user-images.githubusercontent.com/49701237/77896004-28a30300-72aa-11ea-9d1a-817c09290b4c.png)
update the picture.

**Comment by 827Dream** [comment]
> ![ls1](https://user-images.githubusercontent.com/49701237/77896004-28a30300-72aa-11ea-9d1a-817c09290b4c.png) update the picture.

hi do you solve it ?

**Comment by 827Dream** [comment]
i try this . but when i use mkdir to create a directory in c partions  it is fail 
![image](https://user-images.githubusercontent.com/52020508/137686987-004820e1-94f0-46a9-b99f-59aab447073d.png)

**Comment by Rainzhouzzz** [comment]
> > ![ls1](https://user-images.githubusercontent.com/49701237/77896004-28a30300-72aa-11ea-9d1a-817c09290b4c.png) update the picture.
> 
> hi do you solve it ?

no,I did not solve it, but your can try another project ,it works well. please try this link:https://github.com/Rainzhouzzz/efifs, I test it.it work well.
